app/services/notifier/sms_notifier.rb

```diff
@@ -10,6 +10,7 @@ def initialize(real_messages)
       end
 
       @logger = Rails.logger
+      # rubocop:disable Style/StringHashKeys
```

Comment:
Just disable this error in the `.rubocop.yml`.  I don't like linter ignore comments in code, and I don't think that this particular rule is useful.

Reply:
I will disagree - besides being generally bad form, strings in ruby can be mutated in-place, which opens up a bunch of problems with using them as keys in hashes. They should really only be used when building literals that are immediately getting marshalled and thrown into an external api, and forcing people to write out disable comments makes them at least think about why they're using string keys.

Reply:
Nevermind, I am wrong about the in-place thing.

I will still defend it being un-rubylike, and bad form though.

Reply:
Because this rule doesn't light up too much code right now, I'm willing to concede this.  I want to avoid the scenario where we are constantly working around our linter though, so we might need to turn it off in the future if we need to pass a lot of string keys to apis.